dateparser/date.py

```diff
@@ -167,7 +160,7 @@ def __init__(self, locale, date_string, date_formats, settings=None):
         if isinstance(date_formats, six.string_types):
             warn(self.DATE_FORMATS_ERROR_MESSAGE, FutureWarning)
             date_formats = [date_formats]
-        elif not (date_formats is None or isinstance(date_formats, (list, tuple, Set))):
+        elif not (date_formats is None or isinstance(date_formats, (list, tuple, set))):
```

Comment:
It's not exactly the same. `Set` would include subclasses of `collections.abc.Set` different than `set`, but I agree that it's better to simplify here, as another implementation shouldn't be used here.